Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -117,28 +117,49 @@ defineSuite([
         });
     });
 
+    it('rejects readyPromise with invalid tileset version', function() {
+        var tilesJson = {
+            "asset" : {
+                "version" : "2.0"
+            }
```

Comment:
I left the strings on purpose just like the CZML examples in Sandcastle, and the glTF examples for testing in gltf-pipeline.  "2.0" is supposed to be a string just like glTF.